src/vivarium/examples/disease_model/disease.py

```diff
@@ -168,7 +173,9 @@ def setup(self, builder: Builder) -> None:
             source=lambda index: pd.Series(cause_specific_mortality_rate, index=index),
         )
         builder.value.register_value_modifier(
-            "mortality_rate", modifier=self.delete_cause_specific_mortality
+            "mortality_rate",
+            modifier=self.delete_cause_specific_mortality,
+            required_resources=[self.cause_specific_mortality_rate],
```

Comment:
lol, ok now I've lost context and I'm confused. Remind me how the dependencies were assigned before? I thought it was the default_config dict but I don't see that in disease.py to begine with...

Reply:
It was passing before just by random chance.